drivers/platform/xilinx/spi_extra.h

```diff
@@ -98,4 +98,24 @@ typedef struct xil_spi_desc {
 	void			*instance;
 } xil_spi_desc;
 
-#endif // SPI_EXTRA_H_
\ No newline at end of file
+/**
+ * @brief Xilinx platform specific SPI platform ops structure
+ */
+const struct spi_platform_ops xil_platform_ops;
```

Comment:
Does this declaration work good with the definition in the .c file? I think the struct should either be defined here or just defined in the .c file and everyone that wants to use it should declare an `extern const struct spi_platform_ops xil_platform_ops;`.